### ISSUE TITLE
Is there a way to import existing github issues into linear via sync?

### ISSUE THREAD

**Issue by ShravanSunder** [jtormey/linear-sync#35]
i created a new sync connection
- new issues are being synced and have the same data
- however existing issues do not sync to linear

What should i do to sync existing github issues into linear and keep them in sync?

**Comment by janbaykara** [comment]
Agreed with this. I would add two checkboxes to the `New Issue Sync` setup view:

- [ ] Import all Github issues to Linear now
- [ ] Import all Linear issues to Github now

This would configure them for syncing in the same way as for newly created issues.

**Comment by janbaykara** [comment]
Similarly, if one has already used the [Linear importer for Github](https://linear.app/docs/import-issues#github-issues), then linear-sync.com should handle syncing for these issues. This could potentially be another checkbox:

- [ ] Enable syncing for issues manually imported to Linear from Github using[the official importer](https://linear.app/docs/import-issues#github-issues)

The useful thing about official imported issues is they should have metadata identifying which Github issue they relate to, via [IssueAttachments](https://developers.linear.app/docs/graphql/attachments#:~:text=Attachments%20%2D%20Linear%20API&text=Issue%20attachments%20allow%20you%20to,for%20upcoming%20integrations%20inside%20Linear.). (See also #47)
